tests/acceptance/features/apiSpaces/tusUpload.feature

```diff
@@ -32,3 +32,11 @@ Feature: upload resources using TUS protocol
     And for user "Alice" the space "Project Jupiter" should not contain these entries:
       | upload.txt |
 
+
+  Scenario: upload the same file after renaming the first one
+    Given user "Alice" has uploaded a file with content "uploaded content" to "/upload.txt" via TUS inside of the space "Alice Hansen"
```

Comment:
Any reasons behind not using step like this?
```
  Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"
```

Reply:
I think the bug is only triggered when using TUS

Reply:
Yes, I could reproduce only using TUS upload